26-register_registration/275-creator_navigates_to_files_and_license_tab.feature

```diff
@@ -31,15 +28,24 @@ Feature: Creator navigates to Files and License tab
       | https://github.com/BIBSYSDEV/NVA-Frontend |
       | https://www.nrk.no/                       |
 
+  @TEST_NP-22030
+  @test
   Scenario: Creator marks that a Resource has no File or Linked Resource
     Given Creator navigates to Files and License tab
     When they wish to mark that a Resource have no File or Linked Resource
     Then they see a warning message that the Resource will have no File or Linked Resource
     And they see they can cancel marking the Resource
-    And they see they can confirm marking the Resource
+
+  @TEST_NP-22031
+  @test
+  Scenario: Creator marks file for storage, not publishing
+
+    Given Creator navigates to Files and License tab
+    When upload a File
+    And they mark the File with Administrative Agreement
+    Then they File is not presented on the Landing Page

```

Comment:
```suggestion
    Given Creator navigates to Files and License tab
    When they upload a File
    And they mark the File with Administrative Agreement
    Then the File is not presented on the Landing Page

```

Reply:
Eirik har oppdatert 
`mark the File with Administrative Agreement`-scenario til
`Creator marks file for storage, not publishing`

"Administrative Agreement" er berre ein (av mange grunner) til at ein fil skal lagres, men ikkje publiseres

Reply:
linje 44: And they mark the File for storage